lms/djangoapps/program_enrollments/admin.py

```diff
@@ -102,5 +106,45 @@ class ProgramCourseEnrollmentAdmin(admin.ModelAdmin):
     raw_id_fields = ('program_enrollment', 'course_enrollment')
 
 
+def _pending_role_assignment_enrollment_id(pending_role_assignment):
```

Comment:
I understand according to our logic, all these enrollments should be `waiting_enrollments`. But I think that's too limiting. Why not just say `_role_assignment_enrollment_id`?